src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Service/SecondFactorMailService.php

```diff
@@ -91,6 +109,8 @@ public function __construct(
         $this->emailVerificationUrlTemplate = $emailVerificationUrlTemplate;
         $this->emailTemplateService = $emailTemplateService;
         $this->fallbackLocale = $fallbackLocale;
+        $this->logger = $logger;
+        $this->warnOnMissingTemplateConfiguration = $warnOnMissingTemplateConfiguration;
```

Comment:
Would be nice to check this is in fact a boolean (an exception would point to a bug elsewhere). :+1: otherwise